pylint_protobuf/transform.py

```diff
@@ -51,15 +50,14 @@ def transform_enum(desc):
         names.append((name, astroid.extract_node('{} = {}'.format(name, number))))
     return [(cls_def.name, cls_def)] + names
 
-def inner_types(desc, depth=0):
-    # type: (Descriptor, int) -> str
+def inner_types(desc):
+    # type: (Descriptor) -> str
     fragments = []
     for enum_desc in desc.enum_types:
-        fragments.append(_template_enum(enum_desc, depth=depth+1))
+        fragments.append(_template_enum(enum_desc))
     for inner_desc in desc.nested_types:
-        fragments.append(_template_message(inner_desc, depth=depth+1))
-    return '\n'.join(fragments)
-
+        fragments.append(_template_message(inner_desc))
+    return textwrap.indent('\n'.join(fragments), '    ')
```

Comment:
Here is where we perform the relative indent. multi-nested messages will each apply their own relative indent of inner types, and the top-level class will be well formed.